node/send_receive/tests/eventdata_test.js

```diff
@@ -8,91 +8,79 @@ var chai = require('chai');
 chai.should();
```

Comment:
Should we also consider creating a `unit` folder for unit tests, in parallel of the new `integration` folder?